Project.toml

```diff
@@ -1,7 +1,35 @@
 name = "SNaQ"
 uuid = "c2bf7a07-44d2-4d11-b5e2-76dc5aa199d6"
 authors = ["Claudia Solis-Lemus <crsl4@users.noreply.github.com>", "Paul Bastide <pbastide@users.noreply.github.com>", "Cecile Ane <cecileane@users.noreply.github.com>", "and contributors"]
-version = "0.1.0"
+version = "1.0.0"
+
+[deps]
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+NLopt = "76087f3c-5699-56af-9a33-bf431cd00edd"
+PhyloNetworks = "33ad39ac-ed31-50eb-9b15-43d0656eaa72"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-julia = "1.5"
+CSV = "0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10"
+DataFrames = "1.3"
+DataStructures = "0.9, 0.10, 0.11, 0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.18"
+Distributions = "0.25.115"
+NLopt = "0.5.1, 0.6, 1"
+StaticArrays = "0.8.3, 0.9, 0.10, 0.11, 0.12, 1.0"
+StatsBase = "0.26, 0.27, 0.28, 0.29, 0.30, 0.31, 0.32, 0.33, 0.34"
+julia = "1.11"
```

Comment:
Why require julia v1.11? Would SNaQ fail on julia v1.5 (previous requirement)? Julia v1.11 is so new, I wonder what feature specific to julia v1.11 might be used, that would preclude compatibility with earlier julia versions.

Reply:
v1.11 was previously required because it was (to my knowledge) the only way to set a package's download location to be a specific branch of a GitHub repo instead of the repo's main branch, so this should be able to be reverted to v1.5, but I will run the tests on that version first to verify